### ISSUE TITLE
[feature] github themed preview

### ISSUE THREAD

**Issue by philiparvidsson** [joemccann/dillinger#605]
it would be nice if there was a github themed preview so I don't have to save to github to preview (that looked identical to how github renders markdown)  certain things. or even better, convince github to use your the theme you use on dillinger.io. ;-)

**Comment by joemccann** [comment]
Yeah I've thought about that. If you want to create the stylesheet for the preview (the right hand pane) to match GitHub I'm happy to accept that Pull Request.

**Comment by joemccann** [comment]
@philiparvidsson any update on this?

**Comment by philiparvidsson** [comment]
@joemccann sorry - no updates. I started working on it back when you replied originally but since then I haven't really bothered with it (back then I was doing lots of github work through my studies, not so much anymore since I finished my studies and now work full time)

I noted a few things though - even the character translation is slightly different. for example (iirc) dillinger.io translates --- (triple dash) into em-dash, while github doesn't, so I'm thinking a css configuration wouldn't be enough (or maybe that can be done with css today!?)

I also prefer dillinger.io's look to github, but that's a different story.
